src/apps/search-result/search-result.tsx

```diff
@@ -58,11 +60,18 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
   const { facets: campaignFacets } = useGetFacets(q, filters);
 
   // If q changes (eg. in Storybook context)
-  //  then make sure that we reset the entire result set.
+  // then make sure that we reset the entire result set.
   useEffect(() => {
     setResultItems([]);
   }, [q, pageSize, filters]);
 
+  const { track } = useStatistics();
+  useEffect(() => {
+    track("click", { id: 10, name: "On site search string", trackedData: q });
+    // We actaully just want to track if the query changes.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [q]);
```

Comment:
```
    // We actaully just want to track if the query changes.
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [q]);
```

This gets a bit repetitive to look at.

I wonder if it would be a problem to just include `track` in the list of dependencies. Another option might be to move `const { track } = useStatistics();` into the `useEffect()` hook?

Reply:
This is what it looks like with `track` included as a dependency 😅 :
![image](https://user-images.githubusercontent.com/28546954/202421220-f9203820-ac90-4d72-b350-1cd64ee7a1a9.png)